example/cumulus-tf/variables.tf

```diff
@@ -362,7 +362,7 @@ variable "cumulus_process_activity_version" {
 variable "ecs_task_image_version" {
   description = "docker image version to use for Cumulus hello world task"
     type = string
-    default = "1.9.0"
+    default = "2.1.0-alpha"
```

Comment:
This will need to be updated once the final `cumulus-ecs-task` docker image is published.